interface/main/finder/dynamic_finder_ajax.php

```diff
@@ -145,7 +145,7 @@
         } else if ($colname == 'DOB' || $colname == 'regdate' || $colname == 'ad_reviewed' || $colname == 'userdate1') {
             $arow[] = oeFormatShortDate($row[$colname]);
         } else {
-            $arow[] = $row[$colname];
+            $arow[] = xl($row[$colname]);
```

Comment:
oh, lets wait a second in this commit. Does translation make sense here?

Reply:
There is a lot of columns in patient data that contains values of lists and those must translation..

Reply:
Note that lists items need to get special treatment (get converted to title, which is then translated) via the following function:
https://github.com/openemr/openemr/blob/master/library/options.inc.php#L3784

And in this case, though, it appears would need to use integrate layouts module for each entry to really figure out what to do. And would then use this function: https://github.com/openemr/openemr/blob/master/library/options.inc.php#L2436

Reply:
Ok. I thought a title of options is saved in the patient_date table..